draft-ietf-rats-msg-wrap.md

```diff
@@ -274,6 +274,13 @@ Since the collection type is recursive, implementations may limit the allowed de
 
 Although initially designed for the composite Attester use case, the CMW collection can be repurposed for other use cases requiring CMW aggregation.
 
+### CMW Collections' role in composite Attester topology
+
+A CMW Collection's tree structure is not required to be a spanning tree of the a system's composite Attester topology, i.e, it need not be a reflection of domain membership nor domain dependency.
+If a label changes Verifier state beyond a "hint" (e.g., for better Verifier performance or human comprehension), we say that it carries semantic content.
+If a label carries semantic content, the collection itself is a form of evidence that SHOULD be attested.
+When a label carries semantic content that is not bound to other forms of evidence contained in the collection, the collection SHOULD be signed by an attestation key, e.g., by including the collection in a signed EAT {{-rats-eat}}.
```

Comment:
```suggestion
When a label carries semantic content that is not bound to other forms of evidence contained in the CMW, the CMW SHOULD be signed by an attestation key, e.g., by including the CMW in a signed EAT {{-rats-eat}}.
```
Justification: a cmw can be a cmw-record, cmw-collection, or cbor-tag

Reply:
> Justification: a cmw can be a cmw-record, cmw-collection, or cbor-tag

However, the scope here is limited to collection ("CMW Collections' role in composite Attester topology"), so I think using "collection" is OK.